milli/src/search/mod.rs

```diff
@@ -148,13 +148,15 @@ impl<'a> Search<'a> {
         if let Some(sort_criteria) = &self.sort_criteria {
             let sortable_fields = self.index.sortable_fields(self.rtxn)?;
             for asc_desc in sort_criteria {
-                let field = asc_desc.field();
-                if !sortable_fields.contains(field) {
-                    return Err(UserError::InvalidSortableAttribute {
-                        field: field.to_string(),
-                        valid_fields: sortable_fields,
+                // we are not supposed to find any geoPoint in the criterion
+                if let Some(field) = asc_desc.field() {
```

Comment:
Ok, so this line explains why we match an optional here, as the `_geo` criterion does not have a field value (like for `price` or `title`).